Eevee.Sleep.Bot/Controllers/Mongo/DiscordRoleController.cs

```diff
@@ -0,0 +1,76 @@
+using Discord;
+using Eevee.Sleep.Bot.Models;
+using MongoDB.Driver;
+
+namespace Eevee.Sleep.Bot.Controllers.Mongo;
+
+public static class DiscordRoleContoller {
```

Comment:
Split the "role record" and "tracked role" to 2 controllers, so 1 controller only changes data on 1 collection.

Not the case here, but in case there are actions doing data controls on 2 collections, then it'll be another controller.

Reply:
Divided into two parts 👍 
[3068037](https://github.com/RaenonX-PokemonSleep/eevee-sleep-discord-bot/pull/25/commits/3068037382a5a96f513aa2369f8644c895f36072)